packages/icon-button/src/IconButton.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const Variant = {
+  Light: 'light',
+  Dark: 'dark',
+} as const;
+
+type Variant = typeof Variant[keyof typeof Variant];
+
+export { Variant };
+
+interface SharedIconButtonProps {
+  /**
+   * Determines color of `IconButton`. Can be `light` or `dark`.
+   */
+  variant?: Variant;
+
+  /**
+   * Classname applied to `IconButton`.
+   */
+  className?: string;
+
+  /**
+   * Content to appear inside of `IconButton`.
+   */
+  children?: React.ReactNode;
+
+  /**
+   * Determines whether or not `IconButton` is disabled.
+   */
+  disabled?: boolean;
+}
+
+interface LinkIconButtonProps
+  extends HTMLElementProps<'a'>,
+    SharedIconButtonProps {
+  /**
+   * Destination URL, if supplied `IconButton` will render in `a` tags, rather than `button` tags.
+   */
+  href: string;
+}
+
+interface ButtonIconButtonProps
+  extends HTMLElementProps<'button'>,
+    SharedIconButtonProps {
+  href?: null;
+}
+
+type IconButtonProps = LinkIconButtonProps | ButtonIconButtonProps;
+
+const removeButtonStyle = css`
```

Comment:
Is there a reason we don't always apply these styles?

Reply:
We can apply outside of a condition, no problem!

Reply:
👍